CHANGELOG.md

```diff
@@ -0,0 +1,343 @@
+
+The producer version of
+[`Signal.pipe`](#using-signalpipe-instead-of-racsubject),
+**the `SignalProducer.buffer` method can replace replaying** with
+`RACReplaySubject` or any of the `-replay…` methods.
+
+Conceptually, `buffer` creates a (optionally bounded) queue for events, much
+like `RACReplaySubject`, and replays those events when new `Signal`s are created
+from the producer.
+
+For example, to replay the values of an existing `Signal`, it just needs to be
+fed into the write end of the buffer:
+
+```swift
+let signal: Signal<Int, NoError>
+let (producer, sink) = SignalProducer<Int, NoError>.buffer()
+
+// Saves observed values in the buffer
+signal.observe(sink)
+
+// Prints each value buffered
+producer.start(next: { value in
+    println(value)
+})
+```
+
+### Using startWithSignal instead of multicasting
+
+`RACMulticastConnection` and the `-publish` and `-multicast:` operators were
+always poorly understood features of RAC 2. In RAC 3, thanks to the `Signal` and
+`SignalProducer` split, **the `SignalProducer.startWithSignal` method can
+replace multicasting**.
+
+`startWithSignal` allows any number of observers to attach to the created signal
+_before_ any work is begun—therefore, the work (and any side effects) still
+occurs just once, but the values can be distributed to multiple interested
+observers. This fulfills the same purpose of multicasting, in a much clearer and
+more tightly-scoped way.
+
+For example:
+
+```swift
+let producer = timer(5, onScheduler: QueueScheduler.mainQueueScheduler) |> take(3)
+
+// Starts just one timer, sending the dates to two different observers as they
+// are generated.
+producer.startWithSignal { signal, disposable in
+    signal.observe(next: { date in
+        println(date)
+    })
+
+    signal.observe(someOtherObserver)
+}
+```
+
+## Minor changes
+
+### Disposable changes
+
+[Disposables](ReactiveCocoa/Swift/Disposable.swift) haven’t changed much overall
+in RAC 3, besides the addition of a protocol and minor naming tweaks.
+
+The biggest change to be aware of is that **setting
+`SerialDisposable.innerDisposable` will always dispose of the previous value**,
+which helps prevent resource leaks or logic errors from forgetting to dispose
+manually.
+
+### Scheduler changes
+
+RAC 3 replaces the multipurpose `RACScheduler` class with two protocols,
+[`SchedulerType` and `DateSchedulerType`](ReactiveCocoa/Swift/Scheduler.swift), with multiple implementations of each.
+This design indicates and enforces the capabilities of each scheduler using the type
+system.
+
+In addition, **the `mainThreadScheduler` has been replaced with `UIScheduler` and
+`QueueScheduler.mainQueueScheduler`**. The `UIScheduler` types allows operations
+that target the main thread to run _as soon as possible_, replacing RAC 2’s
+`-performOnMainThread` operator—while `QueueScheduler.mainQueueScheduler` can
+be used when ordering is highly important, or for scheduling work at a future
+date.
```

Comment:
Should this be expanded to describe the event delivery semantics more concretely? E.g. `-UIScheduler` attempts to deliver events synchronously if invoked on the main thread.